features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -203,7 +203,7 @@ internal class SessionReplayFeature constructor(
      */
     internal fun stopRecording() {
         if (isRecording.getAndSet(false)) {
-            @Suppress("ThreadSafety") // TODO REPLAY-1861 can be called from worker thread
+            @Suppress("ThreadSafety") // TODO REPLAY-1861 can be called from any thread
```

Comment:
actually these ones are safe as we are forcing to Main thread under the hood in the recorder :)